src/Plugin/StrawberryRunnersPostProcessor/OcrPostProcessor.php

```diff
@@ -377,8 +440,129 @@ protected function hOCRtoMiniOCR($output, $pageid) {
     return $miniocr->outputMemory(TRUE);
   }
 
-}
+  /**
+   * Builds a clean Command string using a File path.
+   *
+   * @param \stdClass $io
+   *    $io->input needs to contain
+   *           \Drupal\strawberry_runners\Annotation\StrawberryRunnersPostProcessor::$input_property
+   *           \Drupal\strawberry_runners\Annotation\StrawberryRunnersPostProcessor::$input_arguments
+   *    $io->output will contain the result of the processor
+   *
+   * @return null|string
+   */
+  public function buildExecutableCommand_checkSearchable(\stdClass $io) {
```

Comment:
WE could make this protected or private. The public interface defines buildExecutableCommand() only

Reply:
Sorry friend, I simply cloned your function and changed name and code so please refactor it (them, because are 2 more functions to build extra commands I added) as you know, really better that I do.